core/lib/json_api_client_extension/has_one_extension.rb

```diff
@@ -0,0 +1,29 @@
+module JsonApiClientExtension::HasOneExtension
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def has_one(attr_name, options = {})
+      class_eval <<-CODE
+        def #{attr_name}_id=(id)
+          # Uncomment when doing refactoring
+          # ActiveSupport::Deprecation.warn(self.class.name + ".#{attr_name}_id= Use relationships instead")
```

Comment:
This should  not be commented?

Reply:
it should be not commented during development, but not for prod. I will add a check.